wordpress-mobile/WordPress-Android
# Adds reminder message to verify email address after registering domain

Fixes #9815 Subtask: Add reminder message to verify email address when domain registration is completed

This PR adds a reminder that you need to verify your email address after registering a domain. It will show a toast message on the **My Site** screen after you go through the Register Domain flow.

Note that I choose to go with the default background colour on the toast message, even though the design shows a dark background. I thought it was best to stay consistent with the other toast messages in the same. Please let me know if this is a wrong assumption and I can correct it.

Also note that this PR contains some commits from this PR #9839 since it requires that PR to work properly.

To test:
1. Install the wasabi build.
2. Switch to a site on a business plan with unclaimed domain credit
3. The CTA should be displayed near the top of the screen, click on "Register Domain"
4. Pick a domain (preferably .blog).
5. Tap on "CHOOSE DOMAIN" and navigate to contact form details.
6. Fill out the your contact details (if they haven't already been prefilled)
7. Click "REGISTER DOMAIN"
8. If successful, the Success screen should be displayed. Click on "NEXT". You should now be returned to the **My SIte** screen
9. Verify that the "Verify your email address..." toast message appears when you return to the **My Site** screen

PS: Follow domain testing guideline and make sure if you purchase a domain to cancel it.

![Screenshots2](https://user-images.githubusercontent.com/897223/57747840-7c0d6200-7694-11e9-99e6-1471b2c78699.png)
